### DISCUSSION TITLE
Bug: "Lines changed" counter on PRs does not update with the rest of the PR page

### DISCUSSION THREAD

**Discussion by SaschaPeukert** [community/community#36663] [Pull Requests]
Whenever I have an open PR and I push a new commit, the site recognizes the new commit without the need to manually refresh - which is great! But the "lines changed" feature ( e.g. "+34 −47" ) does not update to include the lines of the new commit. That one is only accurate once I hit refresh on the page.

**Comment by ossanna16** [comment]
Hey there @SaschaPeukert and thank you for asking a good question! I'm glad you found our community 🙂
